### PR TITLE
ext: fail fast xfs testing

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
@ader1990  add the description

**Comment by chvalean** [comment]
> @ader1990 add the description

this is just to bisect the kernel issue when running this test.
The PR will not be merged in the end, it will be closed/deleted as it's only temporary.

File: Libraries/Azure.psm1 [hunk 1]
```diff
@@ -2363,18 +2363,21 @@ function Check-AzureVmKernelPanic {
...
-              -or $diagFileContent -like '*grub>*') {
-            return $true
+    if (!$KernelLogs) {
+        $bootDiagnosticFile = "$LogDir\$($vm.RoleName)-SSH-Fail-Boot-Logs.txt"
+        $diagStatus = Get-AzureBootDiagnostics -Vm $vm -BootDiagnosticFile $bootDiagnosticFile
+        if ($diagStatus -and (Test-Path $bootDiagnosticFile)) {
+             $KernelLogs = Get-Content $bootDiagnosticFile
         }
     }
+    if ($KernelLogs -like "*Kernel panic - not syncing:*" -or $KernelLogs -like "*RIP:*" `
```
Comment (iamshital) [inline Libraries/Azure.psm1:2377]: Recently we've implemented a way to put text identifiers in one place.
https://github.com/LIS/LISAv2/blob/master/XML/Other/text-identifiers.xml
And fetch it for comparison, from this file.

In your case also, you can do this to detect the call traces.

**Example:** 
```
.
.
.
            <keyword>Successfully reported handler status</keyword> <!-- Azure Linux Agent Version: WALinuxAgent-2.0.16 -->
        </ProvisionComplete>
    </waagent>
    <kernel>
        <Panic>
            <keyword>Kernel panic - not syncing</keyword>
            <keyword>Call Trace</keyword>
            <keyword>rcu_sched self-detected stall on CPU</keyword>
            <keyword>rcu_sched detected stalls on</keyword>
            <keyword>BUG: soft lockup</keyword>
        </Panic>
    </kernel>
```

File: Testscripts/Windows/FILE-SYSTEM-VERIFICATION-TESTS.ps1 [hunk 2]
```diff
@@ -159,7 +187,7 @@ function Main {
             -password $password -TestName $currentTestData.testName
         if ($state -eq "TestRunning") {
             $resultArr += "ABORTED"
-            Write-LogErr "xfstesting.sh is still running after 4 hours!"
+            Write-LogErr "xfstesting.sh is still running..."
```
Comment (iamshital) [inline Testscripts/Windows/FILE-SYSTEM-VERIFICATION-TESTS.ps1:190]: `xfstesting.sh is still running after $timeout minutes!` Seems useful.

**Review by iamshital — CHANGES_REQUESTED** [review]

**Comment by iamshital** [comment]
I just read Chris comment. Ignore my review please.

**Pull request closed without merging**